runtimes/native/src/apu.c

```diff
@@ -126,6 +129,7 @@ void w4_apuTone (int frequency, int duration, int volume, int flags) {
 
     int channelIdx = flags & 0x03;
     int mode = (flags >> 2) & 0x3;
+    const int pan = ((flags >> 4) & 0x3) % 3;
```

Comment:
```suggestion
    int pan = (flags >> 4) & 0x3;
```

Re: const, I should probably do a style pass on the entire code base, but for now let's a least be consistent :grin: